config/main.py

```diff
@@ -1221,16 +1224,18 @@ def replace(ctx, target_file_path, format, dry_run, verbose):
 @config.command()
 @click.argument('checkpoint-name', type=str, required=True)
 @click.option('-d', '--dry-run', is_flag=True, default=False, help='test out the command without affecting config state')
+@click.option('-n', '--non-strict', is_flag=True, default=False, help='ignore validation for changes to tables not YANG covered', hidden=True)
```

Comment:
>'--non-strict' [](http://example.com/codeflow?start=20&length=14)

How about `--ignore-yang-missing` ?

Reply:
updated to `--ignore-more-from-yang-validation`